test/XRP/issued-currency-client-integration.test.ts

```diff
@@ -114,18 +129,22 @@ describe('IssuedCurrencyClient Integration Tests', function (): void {
     )
   })
 
-  it('enableRippling - rippled', async function (): Promise<void> {
+  it('disallowIncomingXrp/allowIncomingXrp - rippled', async function (): Promise<
```

Comment:
```suggestion
  it('allowIncomingXrp - rippled', async function (): Promise<
```

Reply:
I think it's more clear to just indicate which method specifically is under scrutiny in this test, even if it involves a `disallowIncomingXrp` call as well.  If I recall, a similar case may already be merged for other flag-pair tests... maybe we can update those as well to just indicate the particular case under test.